app/Spago.hs

```diff
@@ -156,6 +156,7 @@ parser = do
     sourcePaths = CLI.many (CLI.opt (Just . SourcePath) "path" 'p' "Source path to include")
     packageName = CLI.arg (Just . PackageName) "package" "Specify a package name. You can list them with `list-packages`"
     packageNames = CLI.many $ CLI.arg (Just . PackageName) "package" "Package name to add as dependency"
+    replPackageNames = CLI.many $ CLI.opt (Just . PackageName) "package" 'r' "Package name to add to the REPL as dependency"
     passthroughArgs = many $ CLI.arg (Just . ExtraArg) " ..any `purs compile` option" "Options passed through to `purs compile`; use -- to separate"
```

Comment:
Unfortunately there’s a nameclash between `-p/—path` here and the shortname for `--package`. Do you know any conventions or an alternative short name for `--package?` (`’r’` is just there temporarily)

Reply:
How about `d`/`--dependency`?